lib/push.js

```diff
@@ -59,6 +71,24 @@ module.exports = function (log, db) {
     },
 
     /**
+     *  Notifies all devices (except the one who joined) that a new device joined the account
+     *
+     * @param uid
+     * @param deviceName
+     * @param currentDeviceId
+     * @promise
+     */
+    notifyNewDevice: function notifyNewDevice(uid, deviceName, currentDeviceId) {
```

Comment:
Sorry, I don't have a lot of ongoing context on this feature - naming of what exactly?